ckanext/canada/helpers.py

```diff
@@ -1113,3 +1113,7 @@ def max_resources_per_dataset() -> Optional[int]:
     max_resource_count = config.get('ckanext.canada.max_resources_per_dataset', None)
     if max_resource_count:
         return int(max_resource_count)
+
+
+def support_email_address() -> str:
+    return config.get('ckanext.canada.support_email_address')
```

Comment:
since we have the config_declaration we can use `[]` instead of `.get()` here (get indicates that it might fail and return None which is not the case)